src/nixos.rs

```diff
@@ -118,7 +118,12 @@ impl OsRebuildArgs {
             let confirmation = dialoguer::Confirm::new().default(false).interact()?;
 
             if !confirmation {
-                return Ok(());
+                // exit doesn't call destructors
+                drop(out_dir);
+                // This seems to be the only way to exit non-zero
+                // without printing more text to stdout.
```

Comment:
Oh wow i didn't even see that you already reviewed this last week, Github didn't show that to me somehow.

I didn't mean stdout specifically, my wording there is misleading. What i meant was that bail prints an Error message to the terminal. I don't think its appropriate to print an error here, since there is nothing wrong with the code / program. Afaict there is no way to use / extend eyre in a way that allows returning a Report without printing anything to the Terminal.

Reply:
It was my fault, I wrote the review message one week ago but forgot to click on the submit button.

In any case, I think displaying an error message if the exit code != 0 is not that bad.

```
> Apply the config?
no
Error:
   0: User didn't confirm the request

Location:
   src/nixos.rs:137
```